routes/api/contacts.js

```diff
@@ -1,25 +1,73 @@
-const express = require('express')
+const express = require("express");
+const router = express.Router();
+const contactsHandler = require("./contactsHandler");
 
-const router = express.Router()
+router.get("/", async (req, res) => {
+  try {
+    const contacts = await contactsHandler.listContacts();
+    res.status(200).json(contacts);
+  } catch (error) {
+    res.status(500).json({ message: error.message });
+  }
+});
 
-router.get('/', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+router.get("/:id", async (req, res) => {
+  const contactId = req.params.id;
+  try {
+    const contact = await contactsHandler.getById(contactId);
+    if (contact) {
+      res.status(200).json(contact);
+    } else {
+      res.status(404).json({ message: "Not found" });
+    }
+  } catch (error) {
+    res.status(500).json({ message: error.message });
+  }
+});
 
-router.get('/:contactId', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+router.post("/", async (req, res) => {
+  const { name, email, phone } = req.body;
```

Comment:
Dodasz jescze validację poprzez Joi? Ustaw tam żeby te pola oprócz validacji to np tutaj powinny być required.